.github/ci_commit_pins/xla.txt

```diff
@@ -1 +1 @@
-ec3fd75b2d4bbcf62917d0818378aa3a7d0edcb7
+743d9fd4d2311fdd514caa977da2b7f29948258d
```

Comment:
Here is the new one again. Hopefully we can merge the change this time.
96c96ce89d2731846b733054f3cef8915f31a609

Reply:
Thanks @alanwaketan